webapp/src/js/controllers/user-language-modal.js

```diff
@@ -44,10 +44,8 @@ angular.module('inboxControllers').controller('UserLanguageModalCtrl',
     };
 
     $scope.submit = function() {
-      if (!$scope.selectedLanguage) {
-        var err = new Error('No language selected');
-        $log.error(err);
-        return $q.reject(err);
+      if (typeof $scope.selectedLanguage === 'undefined') {
```

Comment:
What if (somehow) it's `null` or `""` or something? I feel like the original code `!$scope.selectedLanguage` is a safer option here.